compose/def-designer.yml

```diff
@@ -14,7 +14,7 @@ services:
       MICROSERVICE_KEYS_DEFINITION_DESIGNER: "${IDAM_KEY_CCD_DEFINITION_DESIGNER}"
 
   ccd-definition-designer-api:
-    image: "${CCD_DEFINITION_DESIGNER_API_USE_LOCAL-hmcts.azurecr.io/}hmcts/ccd-definition-designer-api:${CCD_DEFINITION_DESIGNER_API_TAG:-latest}"
+    image: "${CCD_DEFINITION_DESIGNER_API_USE_LOCAL-hmctspublic.azurecr.io/}ccd/definition-store-api:${CCD_DEFINITION_DESIGNER_API_TAG:-latest}"
```

Comment:
i think it should be `definition-designer-api` image. Does it moved to hmctspublic?